code/game/objects/structures/therapy.dm

```diff
@@ -0,0 +1,586 @@
+			START_PROCESSING(SSslow_process, src)
+			icon_state = "metronome1"
+			anchored = 1
+	else
+		..()
+
+/obj/structure/metronome/process()
+	var/counter = 0
+	var/mob/living/carbon/human/H
+	for(var/mob/living/L in view(3,src.loc))
+		counter++
+		if(ishuman(L))
+			H = L
+	if(counter == 1 && H)
+		if(ticktock == "Tick")
+			ticktock = "Tock"
+		else
+			ticktock = "Tick"
+		H << "<span class='notice'><i>[ticktock]. . .</i></span>"
+		H << 'sound/effects/singlebeat.ogg'
+		if(prob(1))
+			H.cure_all_traumas(cure_type = CURE_SOLITUDE)
+
+/obj/machinery/chakrapod
+	name = "Crystal Therapy Pod"
+	desc = "A state-of-the-art crystal therapy pod. Designed to utilize phoron enhanced quartz crystals to remove mental trauma from the body. Proven to be 100% effective 30% of the time!"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "syndipod_0"
+	density = 1
+	anchored = 1
+
+	use_power = 1
+	idle_power_usage = 60
+	active_power_usage = 10000
+
+	var/mob/living/carbon/occupant
+	var/locked
+	var/obj/machinery/chakraconsole/connected
+
+/obj/machinery/chakrapod/Destroy()
+	if (connected)
+		connected.connected = null
+	return ..()
+
+/obj/machinery/chakrapod/relaymove(mob/user as mob)
+	if (user.stat)
+		return
+	src.go_out()
+	return
+
+/obj/machinery/chakrapod/verb/eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject Crystal Therapy Pod"
+
+	if (usr.stat != 0)
+		return
+	src.go_out()
+	add_fingerprint(usr)
+	return
+
+/obj/machinery/chakrapod/verb/move_inside()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Enter Crystal Therapy Pod"
+
+	if (usr.stat != 0 || locked)
+		return
+	if (src.occupant)
+		usr << "<span class='warning'>The pod is already occupied!</span>"
+		return
+	if (usr.abiotic())
+		usr << "<span class='warning'>The subject cannot have abiotic items on.</span>"
+		return
+	if(locked)
+		user << "<span class='warning'>The pod is currently locked!</span>"
+		return
+	usr.pulling = null
+	usr.client.perspective = EYE_PERSPECTIVE
+	usr.client.eye = src
+	usr.forceMove(src)
+	src.occupant = usr
+	update_use_power(2)
+	src.icon_state = "syndipod_1"
+	for(var/obj/O in src)
+		O.forceMove(get_turf(src))
+	src.add_fingerprint(usr)
+	return
+
+/obj/machinery/chakrapod/proc/go_out()
+	if (!src.occupant)
+		return
+
+	if(locked)
+		occupant << "<span class='notice'>You push against the pod door and attempt to escape. This process will take roughly two minutes.</span>"
+		if(!do_after(occupant, 1200))
+			return
+
+	for(var/obj/O in src)
+		O.forceMove(get_turf(src))
+	if (src.occupant.client)
+		src.occupant.client.eye = src.occupant.client.mob
+		src.occupant.client.perspective = MOB_PERSPECTIVE
+	src.occupant.forceMove(get_turf(src))
+	src.occupant = null
+	update_use_power(1)
+	src.icon_state = "syndipod_0"
+	return
+
+/obj/machinery/chakrapod/attackby(obj/item/weapon/grab/G as obj, user as mob)
+	if ((!( istype(G, /obj/item/weapon/grab) ) || !( ismob(G.affecting) )))
```

Comment:
You're checking the type of the grab here and below. Just do `if (!istype(G) || !ismob(G.affecting))` here and you can step one level to the front on the pyramid.